src/main/java/taxi/dao/CarDaoImpl.java

```diff
@@ -211,10 +211,13 @@ private Driver parseDriverFromResultSet(ResultSet resultSet) throws SQLException
         Long driverId = resultSet.getObject("id", Long.class);
         String name = resultSet.getString("name");
         String licenseNumber = resultSet.getString("license_number");
+        String login = resultSet.getString("login");
+        String password = resultSet.getString("password");
         Driver driver = new Driver();
         driver.setId(driverId);
         driver.setName(name);
         driver.setLicenseNumber(licenseNumber);
+        driver.setLogin(login);
```

Comment:
It seems that you forgot to set the password field in the driver object. Add the following line to set the password: `driver.setPassword(password);`